### PR TITLE
Update available dependencies

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: src/Resources/symfony/manala.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ packages:
         default:    '7.0'
         constraint: '5.4|5.5|5.6|7.0'
     mysql:
-        enabled:    true
+        enabled:    false
```
Comment (ogizanagi) [inline src/Resources/symfony/manala.yml]: Not enabled by default anymore?

Comment (chalasr) [reply src/Resources/symfony/manala.yml]: It isn't on [the upstream](https://github.com/Elao/symfony-standard/blob/master/ansible/group_vars/app.yml#L8) and I agree, node is  not enabled by default here, I think the minimal should be proposed. WDYT?

Comment (ogizanagi) [reply src/Resources/symfony/manala.yml]: Alright 👍

Comment (chalasr) [reply src/Resources/symfony/manala.yml]: `SetupTest` fixtures need to be updated due to this change...

Comment (ogizanagi) [reply src/Resources/symfony/manala.yml]: Also needs a rebase 😅

**Pull request merged**